16-lambda/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM amazon/aws-lambda-nodejs:16
+
+ENV AWS_DEFAULT_REGION us-east-1
+ENV SERVICE_ROOT /service
+ENV SERVICE_USER service
+ENV SHELL bash
+
+ENV AWS_SDK_VERSION 2.585.0
+ENV YARN_VERSION 1.22.15
+
+COPY entrypoint-lambda.sh /
+ADD https://raw.githubusercontent.com/articulate/docker-consul-template-bootstrap/master/install.sh /tmp/consul_template_install.sh
+ADD https://raw.githubusercontent.com/articulate/docker-consul-template-bootstrap/master/wait-for-it.sh /wait-for-it.sh
+
+RUN yum -y install shadow-utils make zip && yum clean all \
+    && npm install -g npm@7 \
```

Comment:
It doesn't look super urgent, but while we're here should we bump to npm@8?

Reply:
We've had some issues with npm@8 and packages we tend to use. You can see more details in this [commit](https://github.com/articulate/docker-articulate-node/commit/0d6449349329f24a5dfd21ec5420963f510fea29)